recipes/flit-core/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "3.9.0" %}
+
+package:
+  name: flit-core
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/f/flit/flit-{{ version }}.tar.gz
+  sha256: d75edf5eb324da20d53570a6a6f87f51e606eee8384925cd66a90611140844c7
+  folder: source
+
+build:
+  script: python -m pip install --no-deps -vv source/flit_core/
+  noarch: python
+  number: 1
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

Comment:
add a run constraint? 

```
  run_constrained:
    - flit {{ version }}
```

Reply:
If you haven't released for flit 3.9.0 that would be ideal....

Reply:
Flit currrnt has an exact pin so I don't think we need to worry about current releases. Yes we should have the run constraint.

Reply:
i see so the package already exists in the other feedstock. got it.